lib/chef/data_collector/messages.rb

```diff
@@ -68,6 +68,7 @@ def self.run_end_message(reporter_data)
           "id"                     => run_status.run_id,
           "message_version"        => "1.0.0",
           "message_type"           => "run_converge",
+          "node"                   => run_status.node,
```

Comment:
another way to put this is to test that the rendered node object looks like a sane node object rather than just testing that the key is there.